mailet/standard/src/main/java/org/apache/james/transport/matchers/HasAttachment.java

```diff
@@ -89,4 +87,8 @@
         
         return null; // no attachment found
     }
+
+    private boolean isAttachmentDisposition(String disposition) {
```

Comment:
Nice spot... 

It works because Message extends Part. However your naming is wrong. We do not check if the Part has attachement but if it is an attachment. Thus I propose **isAttachment**. See upcoming PR for MAILET-107.